### PR TITLE
Add Blank frontend project

### PR DESCRIPTION
**Pull request by geeeeenccc**

### CODE REVIEW

**Comment by VladimirKavalerov** [comment]
I think that PR should be named different. Please rename PR name with description that you have on it

File: frontend/README.md
Comment (VladimirKavalerov) [inline frontend/README.md:1]: So, I think that this file can be moved to the root of repo

File: frontend/public/index.html [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    <!---->
+    <!--
+      Notice the use of %PUBLIC_URL% in the tags above.
+      It will be replaced with the URL of the `public` folder during the build.
+      Only files inside the `public` folder can be referenced from the HTML.
+
+      work correctly both with client-side routing and a non-root public URL.
+      Learn how to configure a non-root public URL by running `npm run build`.
+    -->
+    <title>React App</title>
```
Comment (VladimirKavalerov) [inline frontend/public/index.html:21]: I think that this title can be something like "AI Weather"

File: frontend/src/App.tsx [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import React from "react";
+import { MantineProvider, Button } from '@mantine/core';
+
+export default function App() {
+    return (
+        <MantineProvider theme={{ fontFamily: "Greycliff CF, sans-serif" }}>
+            <Button variant="gradient">Does something</Button>
```
Comment (VladimirKavalerov) [inline frontend/src/App.tsx:7]: For now Instead of Button, place here just a text that says something like "Welcome to Weather AI Website"

**Review by VladimirKavalerov — CHANGES_REQUESTED** [review]
And one more thing to discuss in call - we can add GitHub workflow to test our frontend project, o it will be more secure to merge with main.

File: frontend/public/index.html (hunk 1 above)
Comment (VladimirKavalerov) [reply frontend/public/index.html]: Fixed, in 2be38fc780b3f6406378205338da339dd3e5a2fe

Comment (geeeeenccc) [reply frontend/public/index.html]: Fixed in 2be38fc780b3f6406378205338da339dd3e5a2fe

**Comment by geeeeenccc** [comment]
Fixed in 09bb6e57199b8cc1e2f1b22f99a9fcd59ccaa9f3

File: frontend/README.md
Comment (geeeeenccc) [reply frontend/README.md:1]: Fixed in e423587dea067e65481f32d29259547a25d0f9b0

**Review by VladimirKavalerov — APPROVED** [review]
LGTM, Thanks!

**Pull request merged**